examples/overlay/messages/07_input_ngd.json

```diff
@@ -0,0 +1,62 @@
+{
+    "query_graph": {
+        "nodes": {
+            "type-2 diabetes": {
+                "id": "MONDO:0005148"
+            },
+            "drug": {
+                "category": "biolink:ChemicalSubstance"
+            }
+        },
+        "edges": {
+            "treats": {
+                "subject": "drug",
+                "predicate": "biolink:treats",
+                "object": "type-2 diabetes"
+            }
+        }
+    },
+    "knowledge_graph": {
+        "nodes": {
+            "MONDO:0005148": {"name": "type-2 diabetes"},
+            "CHEBI:6801": {
+                "name": "metformin",
+                "category": "biolink:ChemicalSubstance"
+            },
+            "CHEBI:5441": {
+                "name": "glyburide",
+                "category": "biolink:ChemicalSubstance"
+            }
+        },
+        "edges": {
+            "df87ff82": {
+                "subject": "CHEBI:6801",
+                "predicate": "biolink:treats",
+                "object": "MONDO:0005148"
+            },
+            "5133c100": {
+                "subject": "CHEBI:5441",
+                "predicate": "biolink:treats",
+                "object": "MONDO:0005148"
+            }
+        }
+    },
+    "results": [
+        {
+            "node_bindings": {
+                "type-2 diabetes": [{"id": "MONDO:0005148"}],
+                "drug": [{"id": "CHEBI:6801"}]
+            },
+            "edge_bindings": {
+                "treats": [{"id": "df87ff82"}]
+            }
+        }
+    ],
+    "workflow": {
```

Comment:
This `workflow` key is supposed to be a sibling of `message` rather than `query_graph` and co. That is, it should be one level up in this JSON object hierarchy. We should probably tweak how all of the examples are formatted. All of the other examples provide just a Message, but I think they ought to show a Query/Response, which will allow us to include the workflow specifications.

Reply:
So should I add another level to the json file with `query_graph` wrapped in a `message` field with `workflow` on that level? Assuming that is correct is there anything else I should add?

Reply:
@patrickkwang just an FYI about why this was included: we wanted to demonstrate what the actual input would be when invoking this command (with the QG, KG, and workflow field and all). I thought it helpful to demonstrate how to actually call the command. Of course, @finnagin can change the location of it in the JSON object

Reply:
Yeah, I agree we should have this workflow specification in all of the examples. I will go ahead and update the others.

Reply:
Sorry, I'm actually having second thoughts here. Including the workflow specification in the example JSON files would make it impossible to re-use those examples. What if we instead added the parameters specification to the `examples` field in the individual operation spec. For example:
```yml
examples: 
- input: overlay/messages/07_input_ngd.json
  output: overlay/messages/08_output_ngd.json
  parameters:
    virtual_relation_label: ngd1
    qnode_keys:
    - drug
    - type-2 diabetes
```

Reply:
That makes sense to me. @dkoslicki any misgivings about moving the parameters to the yaml file instead of the example json file?

Reply:
I'm fine with having it be in the yaml instead

Reply:
I believe this is fixed and can be resolved now.